### PR TITLE
OWL-ViT box_predictor inefficiency issue

### LINKED ISSUE #26099: OWL-ViT box_predictor is wildly inefficient since box bias is not precomputed
### System Info

The box_predictor function computes box bias from scratch each time it is called, but box bias does not actually depend on anything besides the shape of the feature map (which depends on the batch size (which can be broadcasted), number of ViT patches, and ViT token dim) and therefore should be precomputed. This is a super simple fix and will result in a >>10x inference speedup. (https://github.com/huggingface/transformers/blob/ce2e7ef3d96afaf592faf3337b7dd997c7ad4928/src/transformers/models/owlvit/modeling_owlvit.py#L1389C35-L1389C35)

### Who can help?

@amyeroberts @pasqualedem @alaradirik 

### Information

- [x] The official example scripts
- [ ] My own modified scripts

### Tasks

- [x] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

1. Add some lines of code to box_predictor which time its overall execution + the execution of each individual line
2. Run one of the inference examples

### Expected behavior

compute_box_bias will take a substantial amount of time (due to the normalize_grid_corner_coordinates call) while the other lines will be negligible

### PR DESCRIPTION
**Pull request by rvv-karma**
I am calculating box_bias at the start while initialization, and storing it in a variable, then reusing it at inference. Also, shifting it to the correct memory location at the interence time.
I also converted the numpy implementation to torch implementation, and verified that both are giving the same results.

Fixes #26099 

@5hadytru @amyeroberts
Can you please review?
I updated the code to the best of my knwoledge. Please let me know if you need any changes. Thanks.

### CODE REVIEW

**Review by amyeroberts — COMMENTED** [review]
Thanks for working on this! 

Could you add the equivalent changes to owlv2, as in #29713 and add a contribution for @nvbinh15 in the commit?

Once that's done, all I need to see are the outputs for the slow model tests to confirm everything still runs as before:

```
RUN_SLOW=1 pytest tests/models/owlvit tests/models/owlv2 -vv 
```

File: src/transformers/models/owlvit/modeling_owlvit.py [hunk 1]
```diff
@@ -1293,39 +1293,34 @@ def __init__(self, config: OwlViTConfig):
...
-        box_coordinates = box_coordinates.reshape(
-            box_coordinates.shape[0] * box_coordinates.shape[1], box_coordinates.shape[2]
-        )
-        box_coordinates = torch.from_numpy(box_coordinates).to(device)
+        box_coordinates = box_coordinates.view(-1, 2)
 
         return box_coordinates
 
-    def compute_box_bias(self, feature_map: torch.FloatTensor) -> torch.FloatTensor:
+    def compute_box_bias(self, num_patches: int) -> torch.Tensor:
```
[the remark this answers is not in the document]
Comment (RVV-karma) [reply src/transformers/models/owlvit/modeling_owlvit.py]: Fixed in 83200135c7c78138a32c4aab84bda20c7fc24b4f commit

[the remark this answers is not in the document]
Comment (RVV-karma) [reply src/transformers/models/owlvit/modeling_owlvit.py]: Fixed in ae0d8bc1ceaf2a01deb1774f25d48bd3dab90a8f commit.
I didn't found any other usage of compute_box_bias, so just added the maxsize as 2

File: src/transformers/models/owlvit/modeling_owlvit.py [hunk 2]
```diff
@@ -1351,7 +1346,8 @@ def box_predictor(
         pred_boxes = self.box_head(image_feats)
 
         # Compute the location of each token on the grid and use it to compute a bias for the bbox prediction
-        pred_boxes += self.compute_box_bias(feature_map)
+        self.box_bias = self.box_bias.to(feature_map.device)
+        pred_boxes += self.box_bias
```
[the remark this answers is not in the document]
Comment (RVV-karma) [reply src/transformers/models/owlvit/modeling_owlvit.py]: Fixed in edd18742fe6cac818a49cb7d87ca281e87f7d934 commit

**Comment by RVV-karma** [comment]
Also, I am not sure whether is need to create a new PR or how to again trigger this PR with updated code.
And also, I think I was not able to correctly add @nvbinh15 as a contributer. If I can do it now, please let me know how can I do it. Ant links will be helpful. Thanks!

**Comment by amyeroberts** [comment]
@RVV-karma Thanks for sharing the test outputs! 

Here's a guide: https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

If you don't have anything else to commit, you can do an empty commit to add the contribution e.g.:
```
git commit --allow-empty -m "Co-authored by: nvbinh15 <binh.pdc01@gmail.com>"
```

You'll need to run `make fix-copies` and `make fixup` to resolve the quality issues

**Comment by RVV-karma** [comment]
Thanks a lot @amyeroberts!
I have made the code consistent in owlv2, and added @nvbinh15 as a contributor.
I'm neither getting any errors in the slow test, nor any quality issues.
Please review it and let me know for any changes.

Also, here is the output from `make fixup`:
```
Checking/fixing examples/flax/image-captioning/run_image_captioning_flax.py examples/flax/language-modeling/run_bart_dlm_flax.py examples/flax/language-modeling/run_clm_flax.py examples/flax/language-modeling/run_mlm_flax.py examples/flax/language-modeling/run_t5_mlm_flax.py examples/flax/question-answering/run_qa.py examples/flax/speech-recognition/run_flax_speech_recognition_seq2seq.py examples/flax/summarization/run_summarization_flax.py examples/flax/text-classification/run_flax_glue.py examples/flax/token-classification/run_flax_ner.py examples/flax/vision/run_image_classification.py examples/pytorch/image-classification/run_image_classification_no_trainer.py examples/pytorch/image-pretraining/run_mim_no_trainer.py examples/pytorch/language-modeling/run_clm_no_trainer.py examples/pytorch/language-modeling/run_mlm_no_trainer.py examples/pytorch/multiple-choice/run_swag_no_trainer.py examples/pytorch/question-answering/run_qa_beam_search_no_trainer.py examples/pytorch/question-answering/run_qa_no_trainer.py examples/pytorch/semantic-segmentation/run_semantic_segmentation_no_trainer.py examples/pytorch/speech-pretraining/run_wav2vec2_pretraining_no_trainer.py examples/pytorch/summarization/run_summarization_no_trainer.py examples/pytorch/text-classification/run_glue.py examples/pytorch/text-classification/run_glue_no_trainer.py examples/pytorch/token-classification/run_ner_no_trainer.py examples/pytorch/translation/run_translation.py examples/pytorch/translation/run_translation_no_trainer.py examples/research_projects/jax-projects/wav2vec2/run_wav2vec2_pretrain_flax.py examples/research_projects/rag-end2end-retriever/lightning_base.py examples/tensorflow/image-classification/run_image_classification.py examples/tensorflow/multiple-choice/run_swag.py examples/tensorflow/question-answering/run_qa.py examples/tensorflow/summarization/run_summarization.py examples/tensorflow/text-classification/run_glue.py examples/tensorflow/text-classification/run_text_classification.py examples/tensorflow/translation/run_translation.py src/transformers/__init__.py src/transformers/audio_utils.py src/transformers/cache_utils.py src/transformers/convert_slow_tokenizer.py src/transformers/generation/configuration_utils.py src/transformers/generation/logits_process.py src/transformers/generation/utils.py src/transformers/integrations/__init__.py src/transformers/integrations/bitsandbytes.py src/transformers/integrations/integration_utils.py src/transformers/integrations/quanto.py src/transformers/modeling_flax_utils.py src/transformers/modeling_tf_utils.py src/transformers/modeling_utils.py src/transformers/models/__init__.py src/transformers/models/auto/__init__.py src/transformers/models/auto/configuration_auto.py src/transformers/models/auto/image_processing_auto.py src/transformers/models/auto/modeling_auto.py src/transformers/models/auto/tokenization_auto.py src/transformers/models/autoformer/modeling_autoformer.py src/transformers/models/bert/modeling_tf_bert.py src/transformers/models/clipseg/modeling_clipseg.py src/transformers/models/codegen/modeling_codegen.py src/transformers/models/cohere/__init__.py src/transformers/models/cohere/configuration_cohere.py src/transformers/models/cohere/modeling_cohere.py src/transformers/models/cohere/tokenization_cohere_fast.py src/transformers/models/encodec/modeling_encodec.py src/transformers/models/funnel/modeling_funnel.py src/transformers/models/fuyu/modeling_fuyu.py src/transformers/models/gemma/configuration_gemma.py src/transformers/models/gemma/convert_gemma_weights_to_hf.py src/transformers/models/gemma/modeling_flax_gemma.py src/transformers/models/gemma/modeling_gemma.py src/transformers/models/gptj/modeling_gptj.py src/transformers/models/idefics/modeling_idefics.py src/transformers/models/llama/modeling_llama.py src/transformers/models/mamba/configuration_mamba.py src/transformers/models/mamba/modeling_mamba.py src/transformers/models/mask2former/modeling_mask2former.py src/transformers/models/maskformer/modeling_maskformer.py src/transformers/models/musicgen_melody/__init__.py src/transformers/models/musicgen_melody/configuration_musicgen_melody.py src/transformers/models/musicgen_melody/convert_musicgen_melody_transformers.py src/transformers/models/musicgen_melody/feature_extraction_musicgen_melody.py src/transformers/models/musicgen_melody/modeling_musicgen_melody.py src/transformers/models/musicgen_melody/processing_musicgen_melody.py src/transformers/models/oneformer/modeling_oneformer.py src/transformers/models/owlv2/modeling_owlv2.py src/transformers/models/owlvit/modeling_owlvit.py src/transformers/models/pvt_v2/__init__.py src/transformers/models/pvt_v2/configuration_pvt_v2.py src/transformers/models/pvt_v2/convert_pvt_v2_to_pytorch.py src/transformers/models/pvt_v2/modeling_pvt_v2.py src/transformers/models/rag/configuration_rag.py src/transformers/models/rag/retrieval_rag.py src/transformers/models/superpoint/__init__.py src/transformers/models/superpoint/configuration_superpoint.py src/transformers/models/superpoint/convert_superpoint_to_pytorch.py src/transformers/models/superpoint/image_processing_superpoint.py src/transformers/models/superpoint/modeling_superpoint.py src/transformers/models/trocr/modeling_trocr.py src/transformers/models/tvp/modeling_tvp.py src/transformers/models/vision_encoder_decoder/modeling_vision_encoder_decoder.py src/transformers/models/whisper/generation_whisper.py src/transformers/models/yoso/modeling_yoso.py src/transformers/optimization.py src/transformers/pipelines/automatic_speech_recognition.py src/transformers/pipelines/base.py src/transformers/pipelines/conversational.py src/transformers/pipelines/document_question_answering.py src/transformers/pipelines/image_to_text.py src/transformers/pipelines/table_question_answering.py src/transformers/quantizers/auto.py src/transformers/quantizers/base.py src/transformers/quantizers/quantizer_bnb_4bit.py src/transformers/quantizers/quantizer_bnb_8bit.py src/transformers/quantizers/quantizer_quanto.py src/transformers/testing_utils.py src/transformers/tokenization_utils_base.py src/transformers/tools/base.py src/transformers/trainer.py src/transformers/trainer_pt_utils.py src/transformers/trainer_seq2seq.py src/transformers/trainer_utils.py src/transformers/training_args.py src/transformers/utils/__init__.py src/transformers/utils/dummy_pt_objects.py src/transformers/utils/dummy_tokenizers_objects.py src/transformers/utils/dummy_torchaudio_objects.py src/transformers/utils/dummy_vision_objects.py src/transformers/utils/fx.py src/transformers/utils/import_utils.py src/transformers/utils/quantization_config.py tests/extended/test_trainer_ext.py tests/fsdp/test_fsdp.py tests/generation/test_utils.py tests/models/albert/test_tokenization_albert.py tests/models/align/test_modeling_align.py tests/models/altclip/test_modeling_altclip.py tests/models/autoformer/test_modeling_autoformer.py tests/models/bark/test_modeling_bark.py tests/models/bart/test_tokenization_bart.py tests/models/barthez/test_tokenization_barthez.py tests/models/bartpho/test_tokenization_bartpho.py tests/models/bert/test_tokenization_bert.py tests/models/bert_generation/test_tokenization_bert_generation.py tests/models/bert_japanese/test_tokenization_bert_japanese.py tests/models/bertweet/test_tokenization_bertweet.py tests/models/big_bird/test_tokenization_big_bird.py tests/models/biogpt/test_tokenization_biogpt.py tests/models/blenderbot_small/test_tokenization_blenderbot_small.py tests/models/blip/test_modeling_blip.py tests/models/blip_2/test_modeling_blip_2.py tests/models/bloom/test_tokenization_bloom.py tests/models/camembert/test_tokenization_camembert.py tests/models/canine/test_tokenization_canine.py tests/models/chinese_clip/test_modeling_chinese_clip.py tests/models/clap/test_modeling_clap.py tests/models/clip/test_modeling_clip.py tests/models/clip/test_tokenization_clip.py tests/models/clipseg/test_modeling_clipseg.py tests/models/clvp/test_modeling_clvp.py tests/models/clvp/test_tokenization_clvp.py tests/models/code_llama/test_tokenization_code_llama.py tests/models/codegen/test_tokenization_codegen.py tests/models/cohere/__init__.py tests/models/cohere/test_modeling_cohere.py tests/models/cohere/test_tokenization_cohere.py tests/models/conditional_detr/test_modeling_conditional_detr.py tests/models/cpmant/test_modeling_cpmant.py tests/models/cpmant/test_tokenization_cpmant.py tests/models/ctrl/test_tokenization_ctrl.py tests/models/deberta/test_tokenization_deberta.py tests/models/deberta_v2/test_tokenization_deberta_v2.py tests/models/detr/test_modeling_detr.py tests/models/distilbert/test_tokenization_distilbert.py tests/models/dpr/test_tokenization_dpr.py tests/models/dpt/test_modeling_dpt_hybrid.py tests/models/electra/test_tokenization_electra.py tests/models/encodec/test_modeling_encodec.py tests/models/ernie_m/test_tokenization_ernie_m.py tests/models/fastspeech2_conformer/test_modeling_fastspeech2_conformer.py tests/models/fastspeech2_conformer/test_tokenization_fastspeech2_conformer.py tests/models/flava/test_modeling_flava.py tests/models/fnet/test_tokenization_fnet.py tests/models/fsmt/test_tokenization_fsmt.py tests/models/funnel/test_tokenization_funnel.py tests/models/gemma/test_tokenization_gemma.py tests/models/gpt2/test_tokenization_gpt2.py tests/models/gpt_neox_japanese/test_tokenization_gpt_neox_japanese.py tests/models/gpt_sw3/test_tokenization_gpt_sw3.py tests/models/gptj/test_modeling_gptj.py tests/models/gptsan_japanese/test_tokenization_gptsan_japanese.py tests/models/groupvit/test_modeling_groupvit.py tests/models/herbert/test_tokenization_herbert.py tests/models/informer/test_modeling_informer.py tests/models/instructblip/test_modeling_instructblip.py tests/models/kosmos2/test_modeling_kosmos2.py tests/models/layoutlm/test_tokenization_layoutlm.py tests/models/layoutlmv2/test_modeling_layoutlmv2.py tests/models/layoutlmv2/test_tokenization_layoutlmv2.py tests/models/layoutlmv3/test_tokenization_layoutlmv3.py tests/models/layoutxlm/test_tokenization_layoutxlm.py tests/models/led/test_tokenization_led.py tests/models/llama/test_tokenization_llama.py tests/models/longformer/test_modeling_longformer.py tests/models/longformer/test_tokenization_longformer.py tests/models/luke/test_tokenization_luke.py tests/models/lxmert/test_tokenization_lxmert.py tests/models/m2m_100/test_tokenization_m2m_100.py tests/models/marian/test_tokenization_marian.py tests/models/markuplm/test_tokenization_markuplm.py tests/models/maskformer/test_modeling_maskformer.py tests/models/mbart/test_tokenization_mbart.py tests/models/mbart50/test_tokenization_mbart50.py tests/models/mgp_str/test_tokenization_mgp_str.py tests/models/mluke/test_tokenization_mluke.py tests/models/mobilebert/test_tokenization_mobilebert.py tests/models/mobilenet_v2/test_modeling_mobilenet_v2.py tests/models/mpnet/test_tokenization_mpnet.py tests/models/mra/test_modeling_mra.py tests/models/musicgen/test_modeling_musicgen.py tests/models/musicgen_melody/__init__.py tests/models/musicgen_melody/test_feature_extraction_musicgen_melody.py tests/models/musicgen_melody/test_modeling_musicgen_melody.py tests/models/musicgen_melody/test_processor_musicgen_melody.py tests/models/mvp/test_tokenization_mvp.py tests/models/nllb/test_tokenization_nllb.py tests/models/nougat/test_tokenization_nougat.py tests/models/openai/test_tokenization_openai.py tests/models/owlv2/test_modeling_owlv2.py tests/models/owlvit/test_modeling_owlvit.py tests/models/pegasus/test_tokenization_pegasus.py tests/models/perceiver/test_tokenization_perceiver.py tests/models/phobert/test_tokenization_phobert.py tests/models/pix2struct/test_modeling_pix2struct.py tests/models/plbart/test_tokenization_plbart.py tests/models/prophetnet/test_tokenization_prophetnet.py tests/models/pvt_v2/__init__.py tests/models/pvt_v2/test_modeling_pvt_v2.py tests/models/qwen2/test_tokenization_qwen2.py tests/models/rag/test_modeling_rag.py tests/models/rag/test_modeling_tf_rag.py tests/models/realm/test_tokenization_realm.py tests/models/reformer/test_tokenization_reformer.py tests/models/rembert/test_tokenization_rembert.py tests/models/roberta/test_tokenization_roberta.py tests/models/roc_bert/test_tokenization_roc_bert.py tests/models/roformer/test_tokenization_roformer.py tests/models/seamless_m4t/test_tokenization_seamless_m4t.py tests/models/seggpt/test_modeling_seggpt.py tests/models/siglip/test_modeling_siglip.py tests/models/siglip/test_tokenization_siglip.py tests/models/speech_to_text/test_modeling_speech_to_text.py tests/models/speech_to_text/test_tokenization_speech_to_text.py tests/models/speech_to_text_2/test_tokenization_speech_to_text_2.py tests/models/speecht5/test_modeling_speecht5.py tests/models/speecht5/test_tokenization_speecht5.py tests/models/squeezebert/test_tokenization_squeezebert.py tests/models/superpoint/__init__.py tests/models/superpoint/test_image_processing_superpoint.py tests/models/superpoint/test_modeling_superpoint.py tests/models/t5/test_tokenization_t5.py tests/models/table_transformer/test_modeling_table_transformer.py tests/models/tapas/test_tokenization_tapas.py tests/models/time_series_transformer/test_modeling_time_series_transformer.py tests/models/udop/test_tokenization_udop.py tests/models/univnet/test_modeling_univnet.py tests/models/vilt/test_modeling_vilt.py tests/models/vit_hybrid/test_modeling_vit_hybrid.py tests/models/vit_mae/test_modeling_vit_mae.py tests/models/vits/test_modeling_vits.py tests/models/vits/test_tokenization_vits.py tests/models/wav2vec2/test_tokenization_wav2vec2.py tests/models/wav2vec2_phoneme/test_tokenization_wav2vec2_phoneme.py tests/models/whisper/test_modeling_whisper.py tests/models/whisper/test_tokenization_whisper.py tests/models/x_clip/test_modeling_x_clip.py tests/models/xglm/test_tokenization_xglm.py tests/models/xlm/test_tokenization_xlm.py tests/models/xlm_prophetnet/test_tokenization_xlm_prophetnet.py tests/models/xlm_roberta/test_tokenization_xlm_roberta.py tests/models/xlnet/test_tokenization_xlnet.py tests/quantization/quanto_integration/__init__.py tests/quantization/quanto_integration/test_quanto.py tests/test_configuration_utils.py tests/test_feature_extraction_utils.py tests/test_image_processing_utils.py tests/test_modeling_common.py tests/test_modeling_tf_utils.py tests/test_modeling_utils.py tests/test_tokenization_common.py tests/test_tokenization_utils.py tests/trainer/test_trainer.py tests/trainer/test_trainer_seq2seq.py tests/utils/test_audio_utils.py utils/check_copies.py utils/check_docstrings.py utils/check_repo.py utils/tests_fetcher.py
297 files left unchanged
python utils/custom_init_isort.py
python utils/sort_auto_mappings.py
python utils/check_doc_toc.py --fix_and_overwrite
running deps_table_update
updating src/transformers/dependency_versions_table.py
python utils/check_copies.py
python utils/check_table.py
python utils/check_dummies.py
python utils/check_repo.py
2024-03-20 14:23:26.474235: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-03-20 14:23:26.474301: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-03-20 14:23:26.475888: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2024-03-20 14:23:27.577134: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Checking all models are included.
Checking all models are public.
2024-03-20 14:23:33.483980: W tensorflow/core/common_runtime/gpu/gpu_bfc_allocator.cc:47] Overriding orig_value setting because the TF_FORCE_GPU_ALLOW_GROWTH environment variable is set. Original config value was 0.
/content/transformers/src/transformers/deepspeed.py:23: FutureWarning: transformers.deepspeed module is deprecated and will be removed in a future version. Please import deepspeed modules directly from transformers.integrations
  warnings.warn(
Checking all models are properly tested.
Checking all objects are properly documented.
Checking all models are in at least one auto class.
Checking all names in auto name mappings are defined.
Checking all keys in auto name mappings are defined in `CONFIG_MAPPING_NAMES`.
Checking all auto mappings could be imported.
Checking all objects are equally (across frameworks) in the main __init__.
Using TensorFlow backend
/content/transformers/src/transformers/generation_flax_utils.py:24: FutureWarning: Importing `FlaxGenerationMixin` from `src/transformers/generation_flax_utils.py` is deprecated and will be removed in Transformers v4.40. Import as `from transformers import FlaxGenerationMixin` instead.
  warnings.warn(
/content/transformers/src/transformers/generation_tf_utils.py:24: FutureWarning: Importing `TFGenerationMixin` from `src/transformers/generation_tf_utils.py` is deprecated and will be removed in Transformers v4.40. Import as `from transformers import TFGenerationMixin` instead.
  warnings.warn(
/content/transformers/src/transformers/generation_utils.py:24: FutureWarning: Importing `GenerationMixin` from `src/transformers/generation_utils.py` is deprecated and will be removed in Transformers v4.40. Import as `from transformers import GenerationMixin` instead.
  warnings.warn(
No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
Checking the DEPRECATED_MODELS constant is up to date.
python utils/check_inits.py
python utils/check_config_docstrings.py
python utils/check_config_attributes.py
python utils/check_doctest_list.py
python utils/update_metadata.py --check-only
2024-03-20 14:23:56.086516: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-03-20 14:23:56.086572: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-03-20 14:23:56.088011: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2024-03-20 14:23:57.301061: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
python utils/check_task_guides.py
python utils/check_docstrings.py
2024-03-20 14:24:05.777878: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-03-20 14:24:05.777951: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-03-20 14:24:05.779878: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2024-03-20 14:24:07.453961: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
/content/transformers/src/transformers/deepspeed.py:23: FutureWarning: transformers.deepspeed module is deprecated and will be removed in a future version. Please import deepspeed modules directly from transformers.integrations
  warnings.warn(
2024-03-20 14:24:38.699831: W tensorflow/core/common_runtime/gpu/gpu_bfc_allocator.cc:47] Overriding orig_value setting because the TF_FORCE_GPU_ALLOW_GROWTH environment variable is set. Original config value was 0.
python utils/check_support_list.py
```

**Review by amyeroberts — APPROVED** [review]
Amazing work - thanks again for adding this and improving our models! 🔥

**Pull request merged**